### ISSUE TITLE
Object reference error while importing Attributes	

### ISSUE THREAD

**Issue by v22arvind** [MscrmTools/MsCrmTools.Translator#23]
Log file has below error while importing translations:
Error	Attributes	Object reference not set to an instance of an object.

No attribute translations are loaded due to this error

**Comment by MscrmTools** [comment]
Well, that will be difficult to help you with so few information...
Can you provide your translation file?

**Comment by v22arvind** [comment]
The error was due to the fact that an attribute (or more) which is present in the import file is missing in CRM. I believe it got deleted as part of some CR while our translators were working on the files for last 2 months.
Due to urgency to complete my import work, I downloaded the code and put few checks in place to avoid this error. After that I got few more errors during relation import, etc. I am correcting those too. I'll raise a pull request to help you find the places where the code is failing for me and the fix. I would have written a details reply with screenshots but I really need to complete this massive translation import in next few days. Hopefully once I raise the pull request, it would help others.
